composer.json

```
@@ -21,7 +21,7 @@
     "require": {
         "php": "^8.0",
         "laravel/framework": "^8.70|^9.0|^10.0|^11.0",
-        "phpunit/phpunit": "^9.5|^10.0",
+        "phpunit/phpunit": "^9.5|^10.0|^11.9",
```

Typo? Did you mean:
```suggestion
        "phpunit/phpunit": "^9.5|^10.0|^11.0",
```
Because 11.9 does not exist at the moment.
https://github.com/sebastianbergmann/phpunit/releases